octoprint_mrbeam/static/js/settings/dev_design_store.js

```diff
@@ -66,15 +103,23 @@ $(function () {
         };
 
         self.changeEnv = function () {
-            if (self.selectedEnv() === 'prod') {
-                self.designStore.DESIGN_STORE_IFRAME_SRC = self.DESIGN_STORE_PRODUCTION_IFRAME_SRC;
-            } else if (self.selectedEnv() === 'staging') {
-                self.designStore.DESIGN_STORE_IFRAME_SRC = self.DESIGN_STORE_STAGING_IFRAME_SRC;
-            } else if (self.selectedEnv() === 'dev') {
-                self.designStore.DESIGN_STORE_IFRAME_SRC = self.DESIGN_STORE_DEVELOPMENT_IFRAME_SRC;
+            if (self.selectedEnv() === self.PROD) {
```

Comment:
There are lots of ifs here I think that is avoidable. We could create a map with the options and the results and just call the map with selectedEnv. What do you think?